### DISCUSSION TITLE
CotEditor.ScriptError

### DISCUSSION THREAD

**Discussion by chkas** [coteditor/CotEditor#1485] [General]
I have the following problem. My script "Run":

~~~
#!/bin/sh
touch ~/xxx
echo hello >&2
~~~

creates the file "xxx", but returns the following error on output to the Console (I would like to have "hello" there)

[2023-05-11 14:57:08] Run
The operation could not be completed. (CotEditor.ScriptError error 0.)

Did I do something wrong? If this works I am super happy with CotEditor.

**Comment by 1024jp** [comment]
Thank you for the post.
This is definitely an error to be fixed.
I'll fix it in the next version.
